batch/library/emr

```diff
@@ -422,6 +422,7 @@ def main():
     name = arguments.pop('name')
     region = arguments.pop('region')
     desired_state = arguments.pop('state')
+    arguments['tags'] = [arguments['tags']]
```

Comment:
If this expects an array, we should just store the `tags` var as an array too.
Otherwise we can't add more than a single tag.

Then we don't even need this line, right?

Reply:
That doesn't work:
TypeError: unhashable type: 'dict'

I couldn't get it to end up as a list where I wanted it without a change to the emr file.

Reply:
So we can only have one tag?
That's a bummer..

Reply:
Basically, not an EdX use case given the way the code looks. At some point we could push something upstream but I'm not up for it right now.